src/renderers/native/ReactNativeTypes.js

```diff
@@ -53,6 +53,19 @@ export type NativeMethodsMixinType = {
   setNativeProps(nativeProps: Object): void,
 };
 
+export interface IReactNativeComponent {
+  blur(): void,
+  focus(): void,
+  measure(callback: MeasureOnSuccessCallback): void,
+  measureInWindow(callback: MeasureInWindowOnSuccessCallback): void,
+  measureLayout(
+    relativeToNativeNode: number,
+    onSuccess: MeasureLayoutOnSuccessCallback,
+    onFail: () => void,
+  ): void,
+  setNativeProps(nativeProps: Object): void,
+}
```

Comment:
Oh, rats! That's unfortunate. 😦 

In that case, then I guess the duplicate type is a reasonable workaround. If we use it, we should just remove the `(ReactNativeComponent.prototype: NativeMethodsMixinType)` cast too I guess.